epub33/core/index.html

```diff
@@ -6573,13 +6573,16 @@ <h4>Introduction</h4>
 					user who has difficulty following the text of a traditional book. Media Overlays also provide a
 					continuous listening experience for readers who are unable to read the text for any reason,
 					something that traditional audio embedding techniques cannot offer. They are even useful for
-					purposes not traditionally considered accessibility concerns (e.g., for language learning or reading
-					of commercial audio books).</p>
+					purposes not traditionally considered accessibility concerns (e.g., for language learning).</p>
 
 				<p>The Media Overlays feature is designed to be transparent to <a>EPUB Reading Systems</a> that do not
 					support the feature. The inclusion of Media Overlays in an EPUB Publication has no impact on the
 					ability of Media Overlay-unaware Reading Systems to render the EPUB Publication as though the Media
 					Overlays are not present.</p>
+				
+					<p>Media Overlays are not an equivalent to audiobooks, as that format is primarily audio-based with text occasionally
+					provided as an alternate format. There is a W3C recommendation for building audio publications called
```

Comment:
Just to nitpick, I wonder if we should be more direct about using this specification, so something like "The W3C [[Audiobooks]] specification is intended for use building audio publications."

But I'm fine either way.

Reply:
Yes, that may sound better, but I let @wareid decide...

Reply:
I like how it sounds too!